src/main/java/Thames/Parser.java

```diff
@@ -0,0 +1,85 @@
+package Thames;
+
+import Thames.command.;
```

Comment:
Do you think we need to explicitly import all 
classes, or is importing the package sufficient?